docs/book/src/multiversion-tutorial/testdata/project/Makefile

```diff
@@ -63,7 +63,7 @@ generate: controller-gen
 
 # Build the docker image
 docker-build: test
-	docker build . -t ${IMG}
+	docker build -t ${IMG} .
```

Comment:
It is not a breaking change. However, just to do a quick poll. 
Do you think that it should be addressed to v2+ or only v3-alpha plugin @estroz @Adirio @gabbifish ?

Reply:
I would say it is a breaking change (v2 projects scaffolded before this PR will have a different Makefile), and thsu, only to `v3-alpha`

Reply:
IMO: `v2 projects scaffolded before this PR will have a different Makefile` does not characterize a breaking change since it will not break the projects scaffolds before this change. However, we have been very restrictive regards changes in v2 and it is not a critical bug fix for the tool which might is a good argumentation to be only v3+. 

+1.

Reply:
Hi @wking, 

Could you please revert the changes in `pkg/plugin/v2` and run `make generate` and push it again. I think would be better to address it only for v3-alpha as well. Could you also add (go/v3-alpha) on the title to make it clear enough? Then, I think would be fine to get this one merged.

Reply:
> ...v2 projects scaffolded before this PR will have a different Makefile...

Ideally there would be a flow to easily re-scaffold projects to pull in minor stuff like this, instead of delaying all of that and batching it in major-version bumps.  Is there a ticket around that I can subscribe to?  If not, should I file one?

> Could you please revert the changes in `pkg/plugin/v2` and...

I think I've dropped the v2 stuff with 1ef003d1 -> cced240f.

Reply:
The philosophy we have had until now is that once scaffolded, kubebuilder doesn't own the file, user can modify them (actually, they will modify files, kubebuilder provides the scheleton but you need to code parts) so once scaffolded we are not allowed to delete a file, and thus, we can't re-scaffold it later. The only exception is that some code is injected into some files with some special markers.

Reply:
> The philosophy we have had until now is that once scaffolded, kubebuilder doesn't own the file, user can modify them...

That offloads a lot of maintenance cost to folks maintaining the consuming project, doesn't it?  E.g. we land this v2 fix, and now every kubebuilder-scaffolded project is on their own to discover, diagnose, and manually patch their own project to pick up the fix for `podman` compatibility.  It seems like there should be a way to maintain kubebuilder input for portions of the scaffolding, and make it easy for downstream maintainers to ask "do the kubebuilder maintainers have any improvements to suggest for my project?", see a set of suggestions, and then accept or reject them as they see fit.  For one approach to this (tracking the scaffolding in a separate branch which is periodically merged into downstream code), see [here][1].  I expect there are many alternative approaches which would also allow for downstream maintainers to delegate scaffolding maintenance to the kubebuilder maintainers.

[1]: https://github.com/openshift/cincinnati-operator/pull/79#issuecomment-728355658